### PR TITLE
Remove the message about jenkinsci-users ML

### PR DESCRIPTION
**Pull request by batmat**
Asking people to simply send this to the ML has FWICT never led to any bugfix. This generally is just sent as is with no details. Better ask people to report issues to avoid spamming the whole users ML.

Last [example](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/jenkinsci-users/57D8NoXeHbQ/My0Uo1-GBwAJ) this morning pushed me to finally propose this PR.

@daniel-beck I think you already talked about removing this in the past.

### CODE REVIEW

**Comment by daniel-beck** [comment]
The one I was considering changing was the "oops" page, but when I reviewed it, the messages actually made sense. It's possible that I was thinking of this message and just didn't remember it was distinct from 'oops'.

**Comment by oleg-nenashev** [comment]
Link to https://wiki.jenkins-ci.org/display/JENKINS/How+to+report+an+issue or its clone on Jenkins.io would be useful

File: src/main/java/hudson/maven/MavenModuleSetBuild.java [hunk 1]
```diff
@@ -924,7 +924,7 @@ protected Result doRun(final BuildListener listener) throws Exception {
                 return Result.FAILURE;
             } catch (RuntimeException e) {
                 // bug in the code.
-                e.printStackTrace(listener.error("Processing failed due to a bug in the code. Please report this to jenkinsci-users@googlegroups.com"));
+                e.printStackTrace(listener.error("Processing failed due to a bug in the code. Please report this to the issue tracker"));
```
Comment (jglick) [inline src/main/java/hudson/maven/MavenModuleSetBuild.java]: Or simply

```java
e.printStackTrace(listener.error("Processing failed unexpectedly"));
```

If the user is motivated enough to capture details and steps to reproduce and so on, they will figure out how to use the issue tracker. Given that we are swamped by vague and unreproducible bugs as it is I am reluctant to make it sound simpler than it is.

**Comment by daniel-beck** [comment]
> Link to https://wiki.jenkins-ci.org/display/JENKINS/How+to+report+an+issue or its clone on Jenkins.io would be useful

https://jenkins.io/redirect/report-an-issue

**Comment by aheritier** [comment]
@batmat can you add the link to https://jenkins.io/redirect/report-an-issue as advised by others ?

**Comment by batmat** [comment]
@aheritier Thanks for the fix, sorry for lagging those days. Seems like there's enough approval and you can merge this? Thanks

**Pull request merged**